### PR TITLE
[Jetcaster] Merge `jetcaster/all_form_factors` into `main`

### PR DESCRIPTION
**Pull request by arriolac**
The new Jetcaster sample now demonstrates how to build an app with Compose such that it supports the following form factors: phones, tablets, foldables, TV and Wear.

readme_cover

### CODE REVIEW

File: Jetcaster/README.md [hunk 1]
```diff
@@ -3,60 +3,34 @@
...
-### Status: 🚧 In progress 🚧
-
-Jetcaster is still in the early stages of development, and as such only one screen has been created so far. However,
-most of the app's architecture has been implemented, as well as the data layer, and early stages of dynamic theming.
-
-
 ## Screenshots
 
-<img src="docs/screenshots.png"/>
+![readme_cover](https://github.com/android/compose-samples/assets/10263978/a58ab950-71aa-48e0-8bc7-85443a1b4f6b)
```
Comment (kul3r4) [inline Jetcaster/README.md:16]: reminder that the Wear one does not have the download function right now, I can upload some screenshots if you like?

File: Jetcaster/README.md [hunk 2]
```diff
@@ -3,60 +3,34 @@
...
 
 ## Features
 
-This sample contains 2 screens so far: the home screen, and a player screen.
+This sample has 3 components: the home screen, the podcast details screen, and the player screen
```
Comment (kul3r4) [inline Jetcaster/README.md:20]: We could potentially have different features for form factors, e.g. Wear does not deal with multiple panels. We can do in another PR.

**Review by kul3r4 — APPROVED** [review]

**Review by jdkoren — APPROVED** [review]

File: Jetcaster/README.md (hunk 2 above)
Comment (arriolac) [reply Jetcaster/README.md:20]: Yes, we can describe the differences of each form factor in more detail. I agree that we should tackle that in a separate PR.

File: Jetcaster/README.md (hunk 1 above)
Comment (arriolac) [reply Jetcaster/README.md:16]: Good call out. I don't think we'll be adding that feature at all, yeah? If so, we can update the screenshot to reflect the actual implementation on Wear.

**Review by chikoski — APPROVED** [review]

**Review by bentrengrove — APPROVED** [review]
